apps/utils/locust_tests/services/server-load/Jenkinsfile

```diff
@@ -44,6 +44,14 @@ spec:
            description: 'BFD Server Load Controller override ami-id. '
                         + 'Defaults to latest server-load AMI from `master`')
 
+    string(name: 'LOCUST_TAGS',
+           description: 'Space-delimited. Run only the locust tasks with ANY of the given @tag(s).  '
+           + 'Will run all tasks if none are provided. Defaults to an empty string.')
```

Comment:
I agree with you. I think this is possible, but I'm not sure how long it would take to implement. I think we can programmatically set the pipeline description inside a Jenkinsfile, right?

So, something like...

```
job_description = sh(script: 'python -c 'import high_volume_suite; high_volume_suite.get_locust_tags()', returnStdout: true).trim()
```

Just not sure if it's worth doing as part of this PR given how long this has already taken